shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1299,7 +1419,7 @@ - (void)viewDidLayoutSubviews {
   _viewportMetrics.physical_height = viewBounds.size.height * scale;
 
   [self updateViewportPadding];
-  [self updateViewportMetrics];
+  [self updateViewportMetricsIfNeeded:NO];
```

Comment:
At the call sites it's not clear what the boolean does (missing named parameters in Objective-C 😞).  How about `-updateViewportMetricsIfNeeded` and `-updateViewportMetricsIfNeededForRotation`?